### ISSUE TITLE
Feature request: get market calendar in various timezones

### ISSUE THREAD

**Issue by nateGeorge** [rsheftel/pandas_market_calendars#42]
I think it would be nice to have the option in `get_calendar()` to specify a timezone you want the calendar in.  Here is some code I've written recently to do this:

```python
def get_eastern_market_open_close():
    ny = pytz.timezone('America/New_York')
    today_ny = datetime.datetime.now(ny)
    ndq = mcal.get_calendar('NASDAQ')
    open_days = ndq.schedule(start_date=today_ny - pd.Timedelta(str(3*365) + ' days'),
                             end_date=today_ny)
    # convert times to eastern
    for m in ['market_open', 'market_close']:
        open_days[m] = open_days[m].dt.tz_convert('America/New_York')

    return open_days
```

I'd be happy to implement this, I don't see any guidelines on collaborating though.  Am I good to just code it and issue a pull request?

Not sure if it's better suited in [`get_calendar`](https://github.com/rsheftel/pandas_market_calendars/blob/201e483f93031b1f8c1a055f97099d6a119438a4/pandas_market_calendars/calendar_utils.py#L42) or [`.schedule`](https://github.com/rsheftel/pandas_market_calendars/blob/8a95494a0d3d7f7acc10ea2903eeeb5a80702a72/pandas_market_calendars/market_calendar.py#L186)

**Comment by rsheftel** [comment]
Thanks for this suggestion. In terms of collaborating I am open and welcoming to all, I just ask that you submit a PR and including tests so that we can make sure in the future any feature you add continues to work with others new PRs.

For this change, I think it makes the most sense to add the "time_zone" as a parameter in the schedule() function and do the conversation there. I think that all the internals should be kept in UTC and the conversation to any specific time zone done at the end point. This will make code maintenance easier.

**Comment by rsheftel** [comment]
@nateGeorge checking in if you want to submit a PR?

**Comment by nateGeorge** [comment]
Sure, but I won't be able to get to it for a week or two probably.  If someone else wants to do it faster that's fine.  Otherwise I'll get to as soon as I can which will probably be 1-2 wks.

**Comment by rsheftel** [comment]
@nateGeorge checking if you would like to submit a PR for this?

**Comment by nateGeorge** [comment]
Whoops, I totally lost track of it.  I could do it this weekend if no one else has done anything with it yet.

**Comment by rsheftel** [comment]
No one has. It's all you

**Comment by rsheftel** [comment]
This is now merged into master from #106 Thanks.
